### PR TITLE
Changing the script to get the exact active controller_id in nvme.py

### PR DESCRIPTION
**Pull request by Naresh-ibm**
currently we are using the 0] string to get the controller id in list_ctrl option. This may not give always the active controller ID of nvme adapter among the two IDs it holds. So changing the nvme option to id-ctrl and finding with exact controller_id string.

also made the script to pep8 compliance.

### CODE REVIEW

**Review by abdhaleegit — APPROVED** [review]
LGTM

File: avocado/utils/nvme.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def get_number_of_ns_supported(controller_name):
     :rtype: integer
     """
     cmd = f"nvme id-ctrl /dev/{controller_name}"
-    out = process.run(cmd, ignore_status=True, sudo=True, shell=True).stdout_text
+    out = process.run(cmd, ignore_status=True, shell=True).stdout_text
```
Comment (PraveenPenguin) [inline avocado/utils/nvme.py:58]: why is this change required? are we not suppose to run this command having root

File: avocado/utils/nvme.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def get_total_capacity(controller_name):
...
```
Comment (PraveenPenguin) [inline avocado/utils/nvme.py:73]: here as well

File: avocado/utils/nvme.py [hunk 3]
```diff
@@ -84,9 +84,9 @@ def get_controller_id(controll_name):
     :param controller_name: Name of the controller eg: nvme0
     :rtype: string
     """
-    cmd = f"nvme list-ctrl /dev/{controll_name}"
-    output = process.system_output(cmd, shell=True, ignore_status=True).decode("utf-8")
+    cmd = f"nvme id-ctrl /dev/{controll_name}"
+    output = process.run(cmd, shell=True, ignore_status=True).stdout_text
```
Comment (PraveenPenguin) [inline avocado/utils/nvme.py:88]: `system_output` is not helping?

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

File: avocado/utils/nvme.py (hunk 3 above)
Comment (Naresh-ibm) [reply avocado/utils/nvme.py:88]: just to reduce the line length I have used this. both will work, I chose this.

File: avocado/utils/nvme.py (hunk 1 above)
Comment (Naresh-ibm) [reply avocado/utils/nvme.py]: > why is this change required? are we not suppose to run this command having root

Done, changed it back

File: avocado/utils/nvme.py (hunk 2 above)
Comment (Naresh-ibm) [reply avocado/utils/nvme.py]: > here as well

Done.

**Review by PraveenPenguin — APPROVED** [review]
LGTM
